src/worker/vdb_upload_worker.py

```diff
@@ -70,9 +71,57 @@ def write_embeddings_to_vector_db(text_embeddings_list, vector_db_metadata, batc
     elif vector_db_metadata.vector_db_type == VectorDBType.LANCEDB:
         upsert_list = create_lancedb_source_chunks(text_embeddings_list, batch_id, job_id, source_filename)
         return write_embeddings_to_lancedb(upsert_list, batch_id)
+    elif vector_db_metadata.vector_db_type == VectorDBType.MONGODB:
+        upsert_list = create_mongodb_source_chunk_dict(text_embeddings_list, batch_id, job_id, source_filename)
+        return write_embeddings_to_mongodb(upsert_list, vector_db_metadata)
     else:
         logging.error('Unsupported vector DB type: %s', vector_db_metadata.vector_db_type.value)
 
+def create_mongodb_source_chunk_dict(text_embeddings_list, batch_id, job_id, source_filename):
+    upsert_list = []
+    for i, (source_text, embedding) in enumerate(text_embeddings_list):
+        upsert_list.append(
+            {"_id": generate_uuid_from_tuple((job_id, batch_id, i)), 
+            "values": embedding, 
+            "source_text": source_text,
+            "source_document": source_filename
+            })
+    return upsert_list
+
+def write_embeddings_to_mongodb(upsert_list, vector_db_metadata):
+    mongo_conn_uri = os.getenv('VECTOR_DB_KEY')
+    try:
+        mongo_client = pymongo.MongoClient(mongo_conn_uri)
+    except Exception as e:
+        logging.error(e)
```

Comment:
Can you alter this logging statement to have a bit more information in it? Something like:
`logging.error('Error connecting to MongoDB via python client: %s', e)`